src/containers/ChampionList/ChampionList.js

```diff
@@ -55,7 +56,13 @@ const ChampionList = () => {
                 <ChampionDetails id={selectedChampionId} />
             </Modal>
             {!isLoading &&
-                <Container>
+                <Container style={{ marginTop: "20px" }}>
+                    <Button
+                        variant="contained" color="primary"
+                        onClick={() => requestSort('name')}
+                    >
+                        {sortConfiguration.direction == 'DESC' ? "ASCENDING" : "DESCENDING"}
```

Comment:
@adriangaro @DanLepsa  I have an error here when, after first render of champions sorted "ASC"/"DESC" when button is pressed, if I reload the page I receive this error:

![Screenshot 2021-04-30 at 13 25 54](https://user-images.githubusercontent.com/29281918/116683184-ea4dab00-a9b7-11eb-88e4-3c2697017acc.png)

Reply:
this happens because your sortConfiguration is null. Either put it like this {sortConfiguration?.direction === 'DESC' ? "ASCENDING" : "DESCENDING"} or make it 'ASCENDING' initially.

Reply:
Added sortConfiguration?.direction 
Thank you.